main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+    "encoding/csv"
+    "github.com/gin-gonic/gin"
+    "io"
+    "log"
+    "os"
+    "strconv"
+)
+
+const CSV_ERROR_MESSAGE = "There was an error when attempting to read the csv file. Please try again later."
```

Comment:
According to Golang best practices, constants shouldn't be all capitalized (as in JAVA).

Could be either PascalCase or camelCase (according to the desired access modified).

Also, this should be part of the main file, a `constants/constant.go` route would be a better place for this kind of messages.